frontend/src/features/options/OptionsAccordion.tsx

```diff
@@ -144,6 +149,7 @@ const OptionsAccordion = () => {
                 onChange={handleChangeShouldRunESRGAN}
               />
             </Flex>
+            <GuideIcon feature={Feature.ESRGAN} />
```

Comment:
You've changed "GFPGAN" to "Face Correction" - probably ESRGAN should just be Upscale for clarity, I believe there are other models/techniques we will bring in at some point.

Reply:
I made that change because GFPGAN is no longer accurate w/ the inclusion of CodeFormer. AFAIK - upscaling is still just ESRGAN. But, for consistency, probably better to abstract away from model names and have it "userspeak" in both